### PR TITLE
[BPK-2938]: Adding title only docs

### PR DESCRIPTION
**Pull request by ojcurtis**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] [Links platform metadata](https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/layouts/links.js)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

**Comment by ojcurtis** [comment]
Screenshot 2019-09-16 at 09 43 26

File: docs/src/pages/WebRatingPage/WebRatingPage.js [hunk 1]
```diff
@@ -171,6 +171,32 @@ const components = [
...
+      </Paragraph>,
+    ],
+    examples: [
+      <div>
+        <BpkRating
+          ariaLabel="9 Excellent would recommend"
+          title="Excellent"
+          value={9}
+        />
+        <br />
```
Comment (shaundon) [inline docs/src/pages/WebRatingPage/WebRatingPage.js]: We really shouldn't be using `<br />` for layout, that's what CSS is for

File: docs/src/pages/WebRatingPage/WebRatingPage.js [hunk 2]
```diff
@@ -171,6 +171,32 @@ const components = [
       </div>,
     ],
   },
+  {
+    id: 'title',
+    title: 'Title only ratings',
+    blurb: [
+      <Paragraph>
+        The rating component&apos;s can support showing only title.
```
Comment (shaundon) [inline docs/src/pages/WebRatingPage/WebRatingPage.js]:
```suggestion
        The rating component can support showing only title.
```

Not an appropriate use of an apostrophe 😉

File: docs/src/pages/WebRatingPage/WebRatingPage.js [hunk 3]
```diff
@@ -171,6 +171,32 @@ const components = [
       </div>,
     ],
   },
+  {
+    id: 'title',
+    title: 'Title only ratings',
```
Comment (shaundon) [inline docs/src/pages/WebRatingPage/WebRatingPage.js]:
```suggestion
    title: 'Title-only ratings',
```

File: docs/src/pages/WebRatingPage/Rating.scss [hunk 4]
```diff
@@ -29,4 +29,8 @@
   &-vertical {
     width: $bpk-spacing-lg * 3;
   }
+
+  &-title-only {
```
Comment (georgegillams) [inline docs/src/pages/WebRatingPage/Rating.scss]: can we make these all use `__` or `--`?
`&-something` just feels DESGUSTANG!

**Pull request merged**